app/src/main/java/org/simple/clinic/remoteconfig/RemoteConfigSync.kt

```diff
@@ -28,7 +27,8 @@ class RemoteConfigSync @Inject constructor(
   override fun syncConfig(): Single<SyncConfig> {
     return Single.just(SyncConfig(
         syncInterval = SyncInterval.FREQUENT,
-        batchSize = BatchSize.SMALL,
-        syncGroup = SyncGroup.FREQUENT))
+        batchSize = 0, // Unused for remote config sync
```

Comment:
Correct me if I am wrong, we are doing this because we want to sync the entire remote config data at once?

Reply:
Not really, what the case is here is the design of the sync framework assumes that anything that is synced has batches, but that is not the case for things like remote config, help, and reports. So the batch size does not matter here because it is never used for remote config sync.